pkg/ddl/executor.go

```diff
@@ -268,6 +290,11 @@ func (e *executor) CreateSchema(ctx sessionctx.Context, stmt *ast.CreateDatabase
 	dbInfo.Charset = chs
 	dbInfo.Collate = coll
 	dbInfo.PlacementPolicyRef = placementPolicyRef
+	dbInfo.IsActiveActive = activeActive
+	dbInfo.SoftDeleteEnable = softDelete
```

Comment:
An active-active database must be soft-delete enabled? Is there any post check to cover illegal cases?